airflow/contrib/hooks/gcp_transfer_hook.py

```diff
@@ -410,15 +431,15 @@ def operations_contain_expected_statuses(operations, expected_statuses):
             with a state in the list,
         :rtype: bool
         """
-        expected_statuses = (
+        expected_statuses_set = (
```

Comment:
Not a fan of placing types in the variable name, is that really needed? Other than that, seems like a hell of a job. Great work!

Reply:
Yes. Because we already have an argument that has the same name.
```
airflow/contrib/hooks/gcp_transfer_hook.py:434: error: Incompatible types in assignment (expression has type "Set[str]", variable has type "Union[Tuple[str], str]")
airflow/contrib/hooks/gcp_transfer_hook.py:442: error: Unsupported operand types for - ("Set[Any]" and "Tuple[str]")
airflow/contrib/hooks/gcp_transfer_hook.py:442: error: Unsupported operand types for - ("Set[Any]" and "str")
airflow/contrib/hooks/gcp_transfer_hook.py:442: note: Right operand is of type "Union[Tuple[str], str]"
```
I do not like names of this type, but I decided that this is a small problem for a local variable.